assets/swg-mini-prompt.css

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2021 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ .swg-mini-prompt-light,
```

Comment:
I think this line has 2 extra left spaces, and the following lines have 6. Could you remove the extra spacing?

Reply:
Whoops, thought the auto formatter would take care of it, but I guess it doesn't handle css files.